tests/service/alert.service.spec.ts

```diff
@@ -118,6 +126,16 @@ describe('Alert service test', () => {
             }));
         }));
 
+        it('should close an alert on timeout correctly', inject([AlertService], (service: AlertService) => {
```

Comment:
It does work because the change I made in closeAlert otherwise test would fail because the alert was not removed from alerts array